### PR TITLE
[BUGFIX] Change sklearn accuracy metric to mxnet accuracy metric

### LINKED ISSUE #886: Self-attentive Sentence Embedding Tutorial Undeclared Dependency (sklearn)
## Description
self-attentive sentence embedding tutorial is using sklearn for accuracy and f1. As we didn't take sklearn as a dependency, we should migrate those usage to the accuracy/f1 in mxnet and drop the dependency in docker

### PR DESCRIPTION
**Pull request by avinashsai**
partially fixes #886

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1209/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1209/2/index.html

**Comment by avinashsai** [comment]
@leezu 
There is some issue with notedown

File: docs/examples/sentiment_analysis/self_attentive_sentence_embedding.md [hunk 1]
```diff
@@ -1,28 +1,58 @@
...
-
-```{.python .input}
+# A Structured Self-attentive Sentence Embedding
+
+After the novelty of word
+embeddings to create new numerical representations of words, natural language
+processing (NLP) has still been effectively improved in many ways. Along with
+the widespread use of embedding techniques, many other methods have been
+developed to further express the semantics and meanings of sentences with words:
+1. A vector representation of multiple words in a sentence can be concatenated
```
Comment (StrayBird-ATSH) [inline docs/examples/sentiment_analysis/self_attentive_sentence_embedding.md:8]: Would you please add a blank line before 1. to fix the indentation problem?
Thanks!

**Comment by avinashsai** [comment]
@StrayBird-ATSH I guess there is a little discrepancy in the notebook presented on the website and the tutorial in the master branch. (https://gluon-nlp.mxnet.io/examples/sentence_embedding/self_attentive_sentence_embedding.html)

For example: heading in the master branch starts with" Training Structured Self-attentive Sentence Embedding", whereas on the website it is "A Structured Self-attentive Sentence Embedding".

**Comment by StrayBird-ATSH** [comment]
> @StrayBird-ATSH I guess there is a little discrepancy in the notebook presented on the website and the tutorial in the master branch. (https://gluon-nlp.mxnet.io/examples/sentence_embedding/self_attentive_sentence_embedding.html)
> 
> For example: heading in the master branch starts with" Training Structured Self-attentive Sentence Embedding", whereas on the website it is "A Structured Self-attentive Sentence Embedding".

Hello Avinash,
The [main website](https://gluon-nlp.mxnet.io/) displays contents from our default branch, also the release branch (currently v0.9.x). For master branch website, please refer to https://gluon-nlp.mxnet.io/master/index.html

**Comment by mli** [comment]
Job PR-1209/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1209/3/index.html

**Comment by avinashsai** [comment]
@StrayBird-ATSH I have made few changes. please review this.

**Comment by avinashsai** [comment]
@leezu can you review this?

**Comment by avinashsai** [comment]
@StrayBird-ATSH Please review this.

**Comment by leezu** [comment]
@avinashsai why is the diff so large? Are you developing on Windows and introducing different line ending characters for example?

**Comment by avinashsai** [comment]
I think it's about notedown output command.
1. jupyter nbconvert notebook.ipynb --to markdown produces the output similar to current tutorial.

2. Whereas notedown input.md > output.ipynb produces output with a different indentation.

I ran 2nd command first as mentioned in gluonnlp website. However difference was very large and indentation is not proper. So I ran 1st command and it produced output similar to current tutorial.
 That's why the difference is large.

**Comment by leezu** [comment]
Many lines are unchanged but do show up as changed. Could you fix the PR so that you only change the actually intended to be changed lines?
Did you verify there is no problem with respect to incompatible Windows line ending characters? You can check autocrfl here https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration

**Comment by avinashsai** [comment]
Fixed PR. Now it clearly shows the changes

**Comment by mli** [comment]
Job PR-1209/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1209/4/index.html

**Comment by avinashsai** [comment]
Iam not sure why the 2 checks failed

**Comment by StrayBird-ATSH** [comment]
> Iam not sure why the 2 checks failed

I have restarted the CI for you. There has planned to be an update on CI to reduce the rate of unexpected failures.

**Comment by avinashsai** [comment]
All checks have passed. Can you review this?

**Comment by StrayBird-ATSH** [comment]
> All checks have passed. Can you review this?

Great! @leezu Please review this. Thanks!

**Comment by avinashsai** [comment]
@leezu please review this

**Pull request merged**